site/src/utils/docs.ts

```diff
@@ -1,4 +1,11 @@
-const DEFAULT_DOCS_URL = "https://coder.com/docs";
+import { getStaticBuildInfo } from "./buildInfo";
+
+function defaultDocsUrl(): string {
+  const docsUrl = "https://coder.com/docs";
+  // If we can get the specific version, we want to include that in default docs URL.
+  const version = getStaticBuildInfo()?.version.split("-")[0];
```

Comment:
```suggestion
  let version = getStaticBuildInfo()?.version;
  const i = version?.indexOf("-") ?? -1;
  if (version.index >= 0) {
    version = version.slice(0, i);
  }
```

idk, this feels slightly clearer to me and avoid allocating an extra array, but I'll leave it up to you if you want to accept this suggestion or not

Reply:
Yeah I agree this is better :)

Reply:
Uh oh compiler error, fixing haha

Reply:
oh, what was I even thinking 🤪 clearly should've been `i` instead of `version.index`